app/gui2/package.json

```diff
@@ -64,6 +64,7 @@
     "@lezer/highlight": "^1.1.6",
     "@noble/hashes": "^1.3.2",
     "@open-rpc/client-js": "^1.8.1",
+    "@tanstack/vue-query": ">= 5.45.0 < 5.46.0",
```

Comment:
What is the reason we're so strict here?

Reply:
We need `react-query` and `vue-query` to depend on the same `query-core`, but I don't think there's any way to specify that to the package manager explicitly. I'll add a note in the file.

Reply:
@kazcw , @farmaazon pnpm yesterday released `catalogs`: https://pnpm.io/catalogs

We might want to migrate to them shortly to make sure that the workspace uses the same version of tanstack-query